lib/boards/elements/output-dmd-display.js

```diff
@@ -64,12 +65,15 @@ class OutputDmdDisplay {
     this.videoOutputPointer = 0;
     this.requestFIRQ = true;
     this.ticksUpdateDmd = 0;
+    this.videoTicksCounter = 0;
+    this.videoCaptureTicks = 0; // When was the reference buffer last captured
```

Comment:
maybe you can update the variable so no comment is needed, like `this.lastVideoCaptureTimeTicks = 0;`?

Reply:
Sure!